tgui/packages/tgui/constants.ts

```diff
@@ -147,7 +147,7 @@ export const RADIO_CHANNELS = [
   },
   {
     name: 'Radio',
-    freq: 1443,
+    freq: 1361, // monkestation edit
```

Comment:
```suggestion
    freq: 1361,
```
no need, radio frequency is monkestation-specific itself anyways